From 8287e3897bdf8ca46cd2b61fbd915dd88e623127 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:54 -0400
Subject: [alsa] mixart: Remove unused variable

Message-id: <140972823573-374-patchset-jkysela@redhat.com>
Patchwork-id: 91070
O-Subject: [RHEL 7.1 PATCH 373/442] ALSA: mixart: Remove unused variable
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

'err' is not used in the function. Remove it.

Signed-off-by: Sachin Kamat <sachin.kamat@samsung.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 330fb10df832b0008892964601d9d142c14a906f)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/mixart/mixart_core.c b/sound/pci/mixart/mixart_core.c
index 71f4bdc..84f6745 100644
--- a/sound/pci/mixart/mixart_core.c
+++ b/sound/pci/mixart/mixart_core.c
@@ -151,13 +151,11 @@ static int send_msg( struct mixart_mgr *mgr,
 {
  u32 headptr, tailptr;
  u32 msg_frame_address;
- int err, i;
+ int i;
 
  if (snd_BUG_ON(msg->size % 4))
   return -EINVAL;
 
- err = 0;
-
  /* get message frame address */
  tailptr = readl_be(MIXART_MEM(mgr, MSG_INBOUND_FREE_TAIL));
  headptr = readl_be(MIXART_MEM(mgr, MSG_INBOUND_FREE_HEAD));
-- 
1.7.1